src/relay/ir/Relay.g4

```diff
@@ -0,0 +1,140 @@
+grammar Relay;
+
+// Lexing
+// comments
+WS : [ \t\n\r]+ -> skip ;
+LINE_COMMENT : '//' .*? '\n' -> skip ;
+COMMENT : '/*' .*? '*/' -> skip ;
+
+// operators
+MUL: '*' ;
+DIV: '/' ;
+ADD: '+' ;
+SUB: '-' ;
+LT: '<' ;
+GT: '>' ;
+LE: '<=' ;
+GE: '>=' ;
+EQ: '==' ;
+NE: '!=' ;
+
+operatorIdent: CNAME ;
+GLOBAL_IDENT: '@' CNAME ;
+LOCAL_IDENT: '%' CNAME ;
+
+MUT: 'mut' ;
+
+BOOL_LIT
+  : 'true'
+  | 'false'
+  ;
+
+// non-negative floats
+FLOAT
+  : INT '.' INT EXP? // 1.35, 1.35E-9, 0.3, 4.5
+  | INT EXP // 1e10 3e4
+  ;
+
+// non-negative ints
+// INT: '0' | [1-9] DIGIT* ; // no leading zeros
+INT: DIGIT+ ;
```

Comment:
seems it will still accept leading zeros.

Reply:
Will it be better to introduce the concept `operator` that is a combination of "+-*/%" and so on symbols with priorities? This would reduce the grammar text.

Reply:
We now intentionally parse accept leading zeros to be more forgiving. I'll delete the old version.